src/sunburst.js

```diff
@@ -167,9 +183,9 @@ export default Kapsule({
         (state.onClick || this.focusOnNode)(d.data);
       })
       .on('mouseover', d => {
-        state.tooltip.style('display', 'inline');
+        state.tooltip.style('display', state.tooltipShow(d.data) ? 'inline' : 'none');
         state.tooltip.html(`<div class="tooltip-title">${
-          getNodeStack(d).map(d => nameOf(d.data)).join(' > ')
+          state.tooltipTitle ? state.tooltipTitle(d) : this._tooltipTitle(d)
```

Comment:
Please keep `_tooltipTitle` nested here in this method. There's no need to make it an external facing class method. It will also make your changes more minimal.

Reply:
Actually I do make use of this in my own use case--I have a need to edit the default title by removing the first element.

Is there a better way I could accomplish this?

Reply:
One way would be to pass the array of names as a second argument to `state.tooltipTitle`. Then the default function can assemble the name from that array, and my use case can do something similar (omitting the first element).

Would that be ok?

Reply:
@vasturiano just bumping this thread.

Reply:
@waxlamp sorry for the hold on this.

Not sure if I'm following though. You shouldn't ever need to call `._tooltipTitle()` from outside the component, since you can format the title any way you wish using `.tooltipTitle(...)`.

Reply:
Ah I think I understand.

Suppose I wanted to recreate the default tooltip text myself, using a custom function. Would I be able to just write my own copy of `getNodeStack()` the same way as in this file, and call that on `d` within my tooltip title callback? If so, then I will do that and make the change you requested here.

Thanks!

Reply:
@vasturiano, I have made the change; please let me know what you think. Thank you!

Reply:
Yes, you got it. Thanks for changing it. 👍